hsctl

```diff
@@ -350,6 +350,7 @@ rebuild_hs() {
     if [ "$2" == "--db" ]; then
```

Comment:
reviewers should have coordinated with Dan Palmer and reviewed his branches/PRs that have now been incorporated to Discover. Do not comment on hsctl here, reach out to Dan if you have further concerns